corehq/apps/export/tests/test_export_views.py

```
@@ -0,0 +1,139 @@
+        cls.user.save()
+
+    @classmethod
+    def tearDownClass(cls):
+        cls.user.delete()
+
+    def setUp(self):
+        self.client.login(username=self.username, password=self.password)
+
+    def tearDown(self):
+        delete_all_export_instances()
+
+    def test_create_form_export(self):
+        resp = self.client.get(
+            reverse(CreateNewCustomFormExportView.urlname, args=[self.domain.name]),
+            {'export_tag': 'my_sweet_xmlns'}
+        )
+        self.assertEqual(resp.status_code, 200)
+
+    def test_create_case_export(self):
+        resp = self.client.get(
+            reverse(CreateNewCustomCaseExportView.urlname, args=[self.domain.name]),
+            {'export_tag': 'random_case'}
+        )
+        self.assertEqual(resp.status_code, 200)
+
+    def test_commit_form_export(self):
+        export_post_data = json.dumps({
+            "doc_type": "FormExportInstance",
+            "domain": "aspace",
+            "xmlns": "http://openrosa.org/formdesigner/237B85C0-78B1-4034-8277-5D37E3EA7FD1",
+            "last_updated": None,
+            "legacy_saved_export_schema_id": None,
+            "is_daily_saved_export": False,
+            "tables": [],
+            "transform_dates": True,
+            "last_accessed": None,
+            "app_id": "6a48b8838d06febeeabb28c8c9516ab6",
+            "is_deidentified": False,
+            "split_multiselects": False,
+            "external_blobs": {},
+            "export_format": "csv",
+            "include_errors": False,
+            "type": "form",
+            "name": "A Villager's Health > Registrationaa > Reg form: 2016-06-27"
+        })
+        resp = self.client.post(
+            reverse(CreateNewCustomFormExportView.urlname, args=[self.domain.name]),
+            export_post_data,
+            content_type="application/json",
+            follow=True
+        )
+        self.assertEqual(resp.status_code, 200)
+        exports = get_form_export_instances('aspace')
```

Looks like the domain submitted against was `"donkeykong"`, but the configuration was created in `"aspace"`, so we probably actually want to raise an error when that happens 😛 . I don't know if that necessarily needs to block release, but for now you could change all the references to `"aspace"` to `self.domain.name`.